test/test_helper.rb

```diff
@@ -0,0 +1,69 @@
+# For get coverage local - uncomment 4-7 lines
+require 'coveralls'
+Coveralls.wear!('rails')
+# SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+#   SimpleCov::Formatter::HTMLFormatter,
+#   Coveralls::SimpleCov::Formatter
+# ]
+ENV['RAILS_ENV'] ||= 'test'
+$:.unshift File.dirname(__FILE__)
+
+require 'rails_app/config/environment'
+require 'rails/test_help'
+require 'devise'
+require 'database_cleaner'
+require 'capybara/rails'
+require 'capybara/dsl'
+require 'capybara-screenshot/minitest'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.